### PR TITLE
bgCol was being called by default

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

**Comment by jordansread** [comment]
review @lindsaycarr ?

File: R/bgCol.R [hunk 1]
```diff
@@ -62,7 +62,7 @@ bgCol.default <- function(col,...){
     y2 <- par("usr")[4]  
   }
   
-  rect(x1,y1,x2,y2,col = col,...)
+  rect(x1,y1,x2,y2,col = col, border = NA, ...)
```
Comment (lindsayplatt) [inline R/bgCol.R:65]: @jread-usgs is this necessary in addition to checking if bg.arg is null?

Comment (jordansread) [inline R/bgCol.R:65]: I guess I wasn't sure why we would want the line even if `bgCol` was called. We want it to do the rect fill, not draw the border too right?

Comment (lindsayplatt) [inline R/bgCol.R:65]: Yeah, it's probably been over plotting the rectangle line on top of the plot frame this whole time

**Comment by lindsayplatt** [comment]
``` r
gs <- gsplot() %>%
    points(1:4, 1:4) %>%
    bgCol(border=NA, col=NULL)
gs

Error in (function (xleft, ybottom, xright, ytop, density = NULL, angle = 45,  : 
  formal argument "border" matched by multiple actual arguments 
```

`border` is coming through as a named argument & also inside `...`

**Comment by jordansread** [comment]
I don't know why border would be a public arg. Should it be used ever?

**Comment by lindsayplatt** [comment]
It shouldn't be used, but if they pass it in, would we want it to fail?

**Pull request merged**